File: c++/Program-5.cpp
//Program-5     :- Write a program to clear the i i-th bit.

#include<iostream>

using namespace std;

int main(){
    int num,pos;
    cin>>num>>pos;

    num=(num &(~(1>>pos)));
    cout<<num<<"\n";

    return 0;
}

File: java/Program_5.java
//Program-5 :- Write a program to clear the i-th bit.
import java.util.Scanner;
public class Program_5 {
    public static void main(String[] args) {
        Scanner sc=new Scanner(System.in);
        System.out.println("Enter a number");
        int n=sc.nextInt();
        System.out.println("Enter bit number");
        int i=sc.nextInt();
        int b=n&(~(1<<i));
        System.out.println("Result is "+b);
    }
}
